contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -197,4 +201,17 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
     return blockedSenders;
   }
+
+  function migratePreviouslyAllowedSenders(address[] memory previousSendersToAdd) external override onlyOwner {
+    require(s_previousToSContract != address(0), "No previous ToS contract");
+    IAccessController previousToSContract = IAccessController(s_previousToSContract);
+    for (uint256 i = 0; i < previousSendersToAdd.length; ++i) {
+      if (
```

Comment:
Gas golf: If you split this into a nested if statement it saves 36 gas. This is because having them together spends the gas to evaluate each of them before doing the boolean logic.